plugin_store/api/utils.py

```diff
@@ -35,3 +36,7 @@ def __init__(self, model: Any = None, *, use_cache: bool = True):
 # noinspection PyPep8Naming
 def FormBody(model: Any = None, *, use_cache: bool = True) -> Any:
     return FormBodyCls(model=model, use_cache=use_cache)
+
+
+class UUID7(UUID1):
```

Comment:
This seems a little odd imo. Whats wrong with uuid.uuid4()

Reply:
v4 are completely random, and DB primary indexes don't like random values. v7 is designed to be monothonic, which is much better suited for indexing. The implementation of v7 generator comes directly from Python 3.14 source code, I just backported it here as 3.14 is not yet released.

This is probably an overkill for announcements, as there shouldn't be many of them in DB at the same time (at least for now as we don't have a way of hiding them without deleting), but it's a good spot to test them out initially before introducing them for plugins.